packages/react-scripts/package.json

```diff
@@ -54,7 +54,7 @@
     "lodash": "^4.17.20",
     "mini-css-extract-plugin": "0.11.3",
     "mkdirp": "^1.0.3",
-    "node-sass": "^6.0.1",
+    "node-sass": "^7.0.0",
```

Comment:
Using the latest will be better because node-sass has been deprecated, and the latest version is `7.0.1`.